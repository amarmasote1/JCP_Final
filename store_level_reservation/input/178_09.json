{
      "startDate" : "<$yesterday>",
      "storeID" : "2092",
      "uom" : "EACH",
      "endDate" : "<$nextWeek>",
      "itemID" : "15345672221"
}
