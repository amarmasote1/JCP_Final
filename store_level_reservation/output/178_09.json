{
    "warnings": [],
    "message": "Data Write Successful",
    "body": null,
    "key": "JCP|2092|15345672221|EACH",
    "status": 212
}