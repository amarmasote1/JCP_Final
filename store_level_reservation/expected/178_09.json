{
	"status": 212,
	"message": "Data Write Successful",
	"key": "JCP|2092|15345672221|EACH",
	"body": null,
	"warnings": []
}